subs/pantry/src/Pantry/Storage.hs

```diff
@@ -147,6 +148,7 @@ Tree
     key BlobId
     cabal BlobId
     cabalType FileType
+    buildType P.BuildFileType
```

Comment:
`buildType` sounds quite confusing to me though I don't have a good alternative other than just `buildFileType` which I also don't like much

Reply:
This column itself is now removed.